3p/iframe-messaging-client.js

```diff
@@ -74,8 +76,8 @@ export class IframeMessagingClient {
    *  @param {Object=} opt_payload The payload of message to send.
    */
   sendMessage(type, opt_payload) {
-    this.hostWindow_.postMessage/*OK*/(
-        serializeMessage(type, this.sentinel_, opt_payload), '*');
+    this.hostWindow_.postMessage/*OK*/(serializeMessage(
+        type, this.sentinel_, opt_payload, this.rtvVersion_), '*');
```

Comment:
I get that. But once receiver confirmed the version number match, `rtvVersion_` is no longer needed right? Or will it perform the check on every message received?  BTW where do we include the check?